.github/workflows/test.yml

```diff
@@ -65,24 +79,33 @@ jobs:
           - '1.4.*'
 
     steps:
-    - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
-    - uses: actions/setup-go@fac708d6674e30b6ba41289acaab6d4b75aa0753 # v4.0.1
-      with:
-        go-version-file: 'go.mod'
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - uses: actions/setup-go@fac708d6674e30b6ba41289acaab6d4b75aa0753 # v4.0.1
+        with:
+          go-version-file: 'go.mod'
 
-    - name: Setup Terraform ${{ matrix.terraform }}
-      uses: hashicorp/setup-terraform@633666f66e0061ca3b725c73b2ec20cd13a8fdd1 # v2.0.3
-      with:
-        terraform_version: ${{ matrix.terraform }}
-        terraform_wrapper: false
+      - name: Setup Terraform ${{ matrix.terraform }}
+        uses: hashicorp/setup-terraform@633666f66e0061ca3b725c73b2ec20cd13a8fdd1 # v2.0.3
+        with:
+          terraform_version: ${{ matrix.terraform }}
+          terraform_wrapper: false
 
-    - name: Run acceptance test
-      run: make testacc
-    - name: Go Coverage
-      uses: gwatts/go-coverage-action@v1.3.0
-      with:
-        coverage-threshold: 5.7 # current (2023.08.03) state, should be increased - ideally up to at least 50% - as soon as possible
-        cover-pkg: ./...
-        ignore-pattern: |
-          /cdp-sdk-go/ 
-          /main.go$
+      - name: Run acceptance test
+        run: make testacc
```

Comment:
This is still running the acceptance tests rather than the unit tests. In a follow up PR (maybe in #46) we should fix that.